packages/ui/src/Code.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC } from 'react'
+import cn from 'classnames'
+
+import styles from './Code.module.scss'
+
+import { PrismLight as SyntaxHighlighter } from 'react-syntax-highlighter'
+
+// Let's hand pick the languages and themes we want to provide:
+// We don't want a large bundle size.
+//
+// Languages:
+import jsx from 'react-syntax-highlighter/dist/esm/languages/prism/jsx'
+import cpp from 'react-syntax-highlighter/dist/esm/languages/prism/cpp'
+import csharp from 'react-syntax-highlighter/dist/esm/languages/prism/csharp'
+import go from 'react-syntax-highlighter/dist/esm/languages/prism/go'
+import java from 'react-syntax-highlighter/dist/esm/languages/prism/java'
+import kotlin from 'react-syntax-highlighter/dist/esm/languages/prism/kotlin'
+import python from 'react-syntax-highlighter/dist/esm/languages/prism/python'
+import scala from 'react-syntax-highlighter/dist/esm/languages/prism/scala'
+
+// Themes
+import vs from 'react-syntax-highlighter/dist/esm/styles/prism/vs'
+import darcula from 'react-syntax-highlighter/dist/esm/styles/prism/darcula'
+
+// Register these builtin languages we'd use frequently
+SyntaxHighlighter.registerLanguage('jsx', jsx)
```

Comment:
I am wondering if we need to setup default languages at all. Every team might have different needs. I would suggest we do not bundle any languages, declare `react-syntax-highlighter` a peer dep and let every team pick their own languages. WDYT?